gitlabform/gitlab/core.py

```diff
@@ -20,12 +20,29 @@
 class GitLabCore:
 
     url = None
-    __token = None
+    token = None
+
+    # This functions sets variable from configuration,
+    # but if key is not set, it uses same variable from environment
+    # in convention "GITLAB_KEYUPPERCASE"
```

Comment:
Can you please add info about this feature to `config.yml` too? It is the main source of information about features and how to configure gitlabform for the users, they should not have to read the code.